config.template

```diff
@@ -1,88 +1,109 @@
 [case]
-# ACME case configuration
-casename = 20160805v0atm.A_WCYCL1850_v0atm.ne30_oEC.edison.alpha7_00
-native_res = ne30
-short_term_archive = 0
-ref_casename_v0 = B1850C5_ne30_v0.4
+
+# casename is a name that identifies the simulation being analyzed.
+casename = casename
+# ref_casename is the name of a reference run to compare agains (or None
+# to turn off comparison with a reference, e.g. if no reference case is
+# available)
+ref_casename = None
 
 [input]
-# names of namelist and streams files
+
+# directory containing model results
+basedir = /dir/to/model/output
```

Comment:
As a heads up, it may be worthwhile to note here that eventually we will have to support the option that model output has been short-term archived. For the time being, short-term archive is off and all model results and files are in one single output directory. When short-term archiving is on, ocean files will go in one directory, sea-ice files in another, so on so forth. The ACME preAndPostProcessing script now has a flag to indicate whether short-term archiving is on or off, so we could use that in the future to define a basedir directory differently for each model component, if short-term archiving is on.